modules/core/include/opencv2/core/base.hpp

```diff
@@ -727,6 +727,8 @@ CV_EXPORTS void setUseIPP(bool flag);
 
 } // ipp
 
+CV_EXPORTS void setUseOptimizations(bool enabled);
```

Comment:
looks like we already have `setUseOptimized(bool)` http://docs.opencv.org/master/modules/core/doc/utility_and_system_functions_and_macros.html?highlight=setuseoptimized#cv2.setUseOptimized